backend/src/api/petition-signature/controllers/petition-signature.js

```diff
@@ -0,0 +1,104 @@
+"use strict";
+
+/**
+ *  petition-signature controller
+ */
+
+const { createCoreController } = require("@strapi/strapi").factories;
+
+const path = require("path");
+
+function joinPaths(...args) {
+    return path.posix.join(...args).replace(':/', '://');
+}
+
+module.exports = createCoreController(
+  "api::petition-signature.petition-signature",
+  ({ strapi }) => ({
+    async add(ctx) {
+      const { petitionId, name, email, questionAnswers } = ctx.request.body;
+      if (!petitionId || !name || !email) {
+        ctx.throw(400, "Missing parameters");
+        return;
+      }
+      const petition = await strapi
+        .query("api::petition.petition")
+        .findOne({ where: { id: { $eq: petitionId } } });
+      if (!petition) {
+        ctx.throw(404, "petition not found");
```

Comment:
Hoofdletter P, zodat het consistent is met de andere messages bij errors die ook met een hoofdletter beginnen